dockers/docker-orchagent/start.sh

```diff
@@ -48,6 +48,7 @@ elif [ "$aboot_platform" == "x86_64-arista_7050_qx32" ]; then
     SWSSCONFIG_ARGS+="td2.32ports.buffers.1.json td2.32ports.buffers.2.json td2.32ports.buffers.3.json "
 elif [ "$onie_platform" == "x86_64-mlnx_x86-r5.0.1400" ]; then
     PORTSYNCD_ARGS+="-p /etc/ssw/ACS-MSN2700/port_config.ini"
+    ORCHAGENT_ARGS+="-p $onie_platform"
```

Comment:
we use getenv to get the onie platform in swss. we do not need to add -p in orchagent prog. https://github.com/Azure/sonic-swss/blob/master/orchagent/qosorch.cpp#L568